README

```diff
@@ -32,7 +30,7 @@ To get good API documentation, run:
 
     $ epydoc -n "PyFCP API manual" -o html fcp
 
-When you install this package (refer INSTALL), you should 
+When you install this package (refer INSTALL), you should
```

Comment:
I’m not opposed to the whitespace changes themselves. Just put them into a separate commit and it’s fine with me. :)